restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResourceV2.java

```diff
@@ -668,7 +655,7 @@ public Response getDocPath(
                     pageStateParam,
                     pageSizeParam,
                     pageSizeParam > 0 ? pageSizeParam : docsApiConfiguration.getSearchPageSize());
-            DocumentDB db = dbFactory.getDocDataStoreForToken(authToken, getAllHeaders(request));
+            db = dbFactory.getDocDataStoreForToken(authToken, getAllHeaders(request));
```

Comment:
Why do we need to call `getDocDataStoreForToken` again? It seems that we already have a `DocumentDB` on line 632 :thinking:

Reply:
oh we probably don't have to anymore...good point

DocumentDB used to get newed up with page size and page state, but now the Paginator handles that